.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "python.pythonPath": "venv/bin/python",
+    "python.linting.pylintEnabled": false,
+    "python.linting.pylamaEnabled": true,
+    "python.linting.enabled": true
+}
```

Comment:
Probably need a new line at the end of the file.

Reply:
it was an auto generated file, so it didn't get one automatically. To save you guys having to reapprove this, I'll just add the newline as an addon to some later PR